src/vmm/src/devices/virtio/block/virtio/io/async_io.rs

```diff
@@ -110,7 +110,6 @@ impl AsyncFileEngine {
         Ok(())
     }
 
-    #[cfg(test)]
```

Comment:
Why remove this? I don't see anything needing access to `file` field from outside this type